tests/ui/test_ui_tools.py

```diff
@@ -1206,38 +1206,6 @@ def test_main_view_content_header_without_header(self, mocker, message,
                 expected_header)
         assert isinstance(view_components[1], Padding)
 
-    @pytest.mark.parametrize('message', [
-        {
-            'type': 'stream',
-            'display_recipient': 'Verona',
-            'stream_id': 5,
-            'subject': 'Test topic',
-            'flags': [],
-            'content': '<p>what are you planning to do this week</p>',
-            'reactions': [],
-            'sender_full_name': 'alice',
-            'timestamp': 1532103879,
-        },
-        {
-            'type': 'private',
-            'sender_email': 'iago@zulip.com',
-            'sender_id': 5,
-            'display_recipient': [{
-                'email': 'AARON@zulip.com',
-                'id': 1,
-                'full_name': 'aaron'
-            }, {
-                'email': 'iago@zulip.com',
-                'id': 5,
-                'full_name': 'Iago'
-            }],
-            'flags': [],
-            'content': '<p>what are you planning to do this week</p>',
-            'reactions': [],
-            'sender_full_name': 'Alice',
-            'timestamp': 1532103879,
-        }
-    ])
     @pytest.mark.parametrize('to_vary_in_each_message', [
```

Comment:
This looks rather strange, since message was a parameter and is now a fixture, but no code is added here.

Not just for this reason, but I'm wondering if a slightly more verbose name for `message` might be good, though these things are difficult to name well.

Reply:
replaced with `message_fixture`.